build.rs

```diff
@@ -21,5 +38,5 @@ fn main() {
 
     println!("cargo:rerun-if-changed=src/capi.rs");
     // if it has been removed
-    println!("cargo:rerun-if-changed=c_examples/libmicrovmi.h");
+    println!("cargo:rerun-if-changed=target/capi/libmicrovmi.h");
```

Comment:
I am not familiar with the `build.rs` workflow, but is this path correct? Shouldn't it be `target/release/capi/libmicrovmi.h`?

Reply:
oh, it's not correct indeed.
i should add 2 lines for target/debug/xxx and target/release/xxx